packages/mapviewer/src/modules/infobox/components/InfoboxContent.vue

```diff
@@ -41,43 +70,83 @@ watch(selectedFeatures, (features) => {
-        >
+        <div class="d-flex h-100 overflow-y-auto justify-content-stretch flex-column flex-md-row">
             <div
                 v-if="showElevationProfile"
                 key="profile-detail"
-                class="h-100 d-flex flex-column flex-md-row align-content-stretch"
+                class="d-flex flex-column align-content-stretch infobox-content"
             >
-                <FeatureElevationProfile
-                    class="flex-grow-1 profile-with-feature"
-                    :animation="animation"
-                />
+                <div
+                    v-if="isMultiFeature && currentGeometryElements.length > 1"
+                    class="ps-1 pt-1 d-flex"
+                >
+                    <div class="d-flex gap-1 mw-100 overflow-x-auto">
+                        <button
+                            v-for="(_, index) in currentGeometryElements"
+                            :key="index"
+                            class="btn btn-sm text-nowrap"
+                            :class="{
+                                'btn-secondary': index === currentGeometryElementIndex,
+                                'btn-light': index !== currentGeometryElementIndex,
+                            }"
+                            :data-cy="`profile-segment-button-${index}`"
+                            @click="currentGeometryElementIndex = index"
+                        >
+                            {{ t('profile_segment', { segmentNumber: index + 1 }) }}
+                        </button>
+                    </div>
+                </div>
+                <GeoadminElevationProfile
+                    :points="currentProfileCoordinates"
+                    :projection="projection.epsg"
+                    :locale="currentLang"
+                    :staging="ENVIRONMENT"
+                    :filename="generateFilename('.csv')"
+                >
+                    <GeoadminElevationProfileCesiumBridge
+                        v-if="is3dActive && getCesiumViewer()"
+                        :cesium-instance="getCesiumViewer()"
+                    />
+                    <GeoadminElevationProfileOpenLayersBridge
+                        v-else-if="olMap"
+                        :ol-instance="olMap"
+                    />
+                </GeoadminElevationProfile>
             </div>
+            <FeatureStyleEdit
+                v-if="isEditingDrawingFeature && showFeatureInfoInBottomPanel"
+                v-show="!showElevationProfile"
+                :feature="selectedFeature"
+                class="drawing-feature-edit p-3"
+                :class="{ 'flex-grow-1': !showElevationProfile }"
+            />
             <FeatureList
                 v-if="showFeatureInfoInBottomPanel"
                 v-show="!showElevationProfile"
             />
         </div>
     </div>
 </template>
+
+<style lang="scss" scoped>
+@import '@/scss/media-query.mixin';
+
+.infobox-content {
```

Comment:
Intentionally not using tailwind here?

Reply:
I thought I shouldn't do a partial switch to TW while in the `mapviewer` package... should we start moving things gradually?

Reply:
good question. Maybe it would be more manageable when doing gradually?